### ISSUE TITLE
TNS Build Issues after installing new version xCode

### ISSUE THREAD

**Issue by NathanaelA** [NativeScript/nativescript-cli#2182]
When upgrading xCode from 8 to 8.1, the `tns` command line started to fail.    No errors; just returned to the command line after spinning for a few minutes.    Running `tns doctor` does not show any failures and says everything is fine.  

 To fix; you need actually open up xCode and answer the question about it needing to download support files.   Once it does download then tns starts working again.   This "error" case should be checked by TNS and then it should alert the user what need to be fixed at least at a minimum with the `tns doctor`.

**Comment by NickIliev** [comment]
Hey @NathanaelA  I have just updated my Xcode from 8 to 8.1 but there was no need at my side to download support files. I was thinking of adding a documentation section on how to handle similar case (open Xcode and follow the prompts) but it seems that the first builds of Xcode were missing those support files but are already included in the latest ones so this should not be reproducible anymore.

**Comment by NathanaelA** [comment]
Actually this will probably happen again with 8.2, 9.0, etc.  

The support files I'm talking about is the xcode command line stuff.  At least on my machine this has always been a separate download that xcode asks about the first time you update xcode when you first start up xcode.   If they are present, then `TNS` fails with no error.   So a simple check would be worthwhile in my opinion.

**Comment by NickIliev** [comment]
@rosen-vladimirov , @dtopuzov ^^^

**Comment by dtopuzov** [comment]
@NickIliev It is a feature, nice to have it, but for sure we have more important thing to solve right now.

We will be happy if someone can contribute, marking with `up-for-grabs`

**Comment by NathanaelA** [comment]
I would disagree; I think it should at a minimum report an error.  Someone who is just starting NativeScript and installed XCode, and NativeScript and doesn't open up xcode is going to wonder why everything says it is fine but nothing works.   This will cause people to leave nativescript because they can't get it working...
